harmony_regridding_service/config/HRS_varinfo_config.json

```diff
@@ -15,17 +15,25 @@
   ],
   "Mission": {
     "ATL\\d{2}": "ICESat2",
-    "SPL4.+": "SMAP"
+    "SPL[34].+": "SMAP"
   },
- "ExcludedScienceVariables": [
+  "ExcludedScienceVariables": [
     {
       "Applicability": {
-        "Mission": "ICESat2"
+	"Mission": "ICESat2"
       },
       "VariablePattern": [
-        "/quality_assessment/.*",
-        "/orbit_info/.*",
-        "/atlas_impulse_response/.*"
+	"/quality_assessment/.*",
+	"/orbit_info/.*",
+	"/atlas_impulse_response/.*"
+      ]
+    },
+    {
+      "Applicability": {
+        "Mission": "SMAP"
+      },
+      "VariablePattern": [
+	".*/freeze_thaw_time_utc.*"
```

Comment:
This just adds the utc string variables to the excluded variables, the code doesn't use it yet, but it will when DAS-(tbd) is writen